Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/TokenHandoffController.java

```diff
@@ -78,9 +77,9 @@ public void doAfterCompose(final Window window) {
                     (HttpServletRequest) Executions.getCurrent().getNativeRequest();
             LOGGER.debug("httpServletRequest: {}", httpServletRequest);
 
-            final String appAuthCookieStr = JwtHelper.readCookie(httpServletRequest, "App_Auth");
```

Comment:
The constants defined in this calss is duplicated in jwtHelper.

Reply:
Removed unused constants from top of class.